manifests/templates/deployments/cf-jobs.yml

```diff
@@ -1,12 +0,0 @@
-jobs:
-  - <<: (( merge ))
-  - name: consul_z1
-    <<: (( merge ))
-    update:
-      max_in_flight: 1
```

Comment:
Somehow this time we are not overriding whole `update` hash but just a value of serial. That's spiff.... you never really know for sure what's it going to do.